Item12391: Fix for SecurityAlert CVE-2013-1666

   Item12391 applies to Foswiki 1.1.0-1.1.7. This patch covers Foswiki 1.1.0-1.1.2
   This patch updates Item12285.

~~~PATCH 5218b3501c96c8e6017f4df81298ef5b:e89b823a46626febb00fe9f0f26ff332 lib/Foswiki/Macros/MAKETEXT.pm (Foswiki-1.1.0,Foswiki-1.1.1,Foswiki-1.1.2)
--- lib/Foswiki/Macros/MAKETEXT.pm	2013-02-13 11:00:18.037967200 -0500
+++ lib/Foswiki/Macros/MAKETEXT.pm	2013-02-13 11:00:39.645714522 -0500
@@ -25,8 +25,8 @@
     $str =~ s/\]/~]/g;
 
     # restore already escaped stuff:
-    $str =~ s/~~\[/~[/g;
-    $str =~ s/~~\]/~]/g;
+    $str =~ s/~~+\[/~[/g;
+    $str =~ s/~~+\]/~]/g;
 
     $max         = 0;
     $min         = 1;

